WalletWasabi.Fluent/Controls/CurrencyEntryBox.axaml.cs

```diff
@@ -373,13 +373,8 @@ private bool TryParsePastedValue(string text, [NotNullWhen(true)] out string? re
 	// Pre-composes the TextInputEventArgs to see the potential Text that is to
 	// be committed to the TextPresenter in this control.
 
-	private string RemoveInvalidCharacters(string? text)
+	private string RemoveInvalidCharacters(string text)
 	{
-		if (text is null)
-		{
-			return "";
-		}
-
```

Comment:
Are you sure that we can do this?
Do to how it is used:
``
	private string PreComposeText(string input)
	{
		input = RemoveInvalidCharacters(input);
		var preComposedText = Text ?? "";
		var caretIndex = CaretIndex;
		var selectionStart = SelectionStart;
		var selectionEnd = SelectionEnd;

		if (!string.IsNullOrEmpty(input) && (MaxLength == 0 ||
''
It's a bit suspicious.

Reply:
For me string.IsNullOrEmpty implies that it might be null, or was it there for no reason?

Reply:
Currently only used here:
'''
protected override void OnTextInput(TextInputEventArgs e)
{
	var input = e.Text == null ? "" : e.Text.TotalTrim();
'''

You are right, this is never null. The code is simply misleading.